kt

```
@@ -47,22 +49,26 @@ parse_args() {
         cliparam[$key]=$value
         ;;
       *)
-        echo "Invalid argument passed: -$opt" >&2
+        echo "Invalid argument passed: -$OPTARG" >&2
+        sub_help
         exit 1
         ;;
     esac
   done
 }
 
 sub_help() {
-    echo "Usage: $progName <subcommand> -e ENVIRONMENT [-e ENVIRONMENT...] [-c COMPONENT] [-p key=value ] [-d]"
+    echo "Usage: $progName <subcommand> -e ENVIRONMENT [-e ENVIRONMENT...] [-c COMPONENT] [-p key=value ] [-d] [-f]"
     echo ""
     echo "Subcommands:"
     echo "    clean     Clean the compile folder (under _build)"
     echo "    compile   Use gomplate to compile the templates"
     echo "    validate  Validate the compiled templates against a Kubernetes API server"
     echo "    deploy    Apply the compiled templates to a Kubernetes API server"
     echo "    delete    Delete the items in the compiled templates on a Kubernetes API server"
+    echo "Flags:"
+    echo "    -d        DEBUG mode"
+    echo "    -f        FORCE mode: ignores errors"
```

could please add to the description and mention that force mode is for the `delete` command only?

---

eg `FORCE mode: ignore errors when trying to delete Kubernetes objects.`